frontend/javascripts/oxalis/shaders/texture_access.glsl.js

```diff
@@ -207,6 +207,14 @@ export const getColorForCoords: ShaderModule = {
 
       float rgbaIndex = linearizeVec3ToIndexWithMod(offsetInBucket, bucketWidth, packingDegree);
 
+      if (packingDegree == 2.0) {
+        if (rgbaIndex == 0.0) {
+          return vec4(bucketColor.r, bucketColor.g, bucketColor.r, bucketColor.g);
```

Comment:
Why is the pattern `rgrg`? Can't wrap my head around it right now :)

Reply:
I added a comment :)